src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/DateTime.cs

```diff
@@ -95,6 +95,30 @@ public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
         }
     }
 
+    // DateTime(year, month, day, hour, minute, second[, millisecond])
+    // Equivalent DAX/Excel function: Time
```

Comment:
> Equivalent DAX/Excel function: Time [](http://example.com/codeflow?start=6&length=36)

Excel Time doesn't accept year, month, day.

Excel has Date, Time just like Fx.
This is equivalent of Date() + Time(). There is a way to achieve this today. 

Is this something requested by customers?

Reply:
Yes, this has been requested for a while.